pandas/tests/io/parser/test_multi_thread.py

```diff
@@ -24,10 +25,18 @@
 ]
 
 
-@xfail_pyarrow  # ValueError: Found non-unique column index
-def test_multi_thread_string_io_read_csv(all_parsers):
+@pytest.mark.filterwarnings("ignore:Passing a BlockManager:DeprecationWarning")
+def test_multi_thread_string_io_read_csv(all_parsers, request):
     # see gh-11786
     parser = all_parsers
+    if parser.engine == "pyarrow":
+        pa = pytest.importorskip("pyarrow")
+        if Version(pa.__version__) < Version("16.0"):
+            request.applymarker(
+                pytest.mark.xfail(
+                    reason="# ValueError: Found non-unique column index", strict=True
```

Comment:
```suggestion
                    reason="# ValueError: Found non-unique column index"
```
(this is already the default)